pkgs/servers/monitoring/grafana-agent/default.nix

```
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "grafana-agent";
-  version = "0.30.2";
+  version = "0.32.1";
 
   src = fetchFromGitHub {
     rev = "v${version}";
     owner = "grafana";
     repo = "agent";
-    sha256 = "sha256-yexCK4GBA997CShtuQQTs1GBsXoknUnWWO0Uotb9EG8=";
+    sha256 = "sha256-t5rQSNLpX0ktK4mKgX6OhNLkANQ1LbFEjmQo/r7UHOM=";
   };
 
-  vendorHash = "sha256-Cl3oygH1RPF+ZdJvkDmr7eyU5daxaZwNE8pQOHK/qP4=";
+  vendorHash = "sha256-TfXVnTM2rG7cvUV5fqD+tglg16scFvwpt0/D92Xvjtg=";
 
```

```suggestion
  vendorHash = "sha256-5beHG1mZkNcDhccleqTlMA+uiV5d6SSh2QWiL4g3O28=";
  proxyVendor = true; # darwin/linux hash mismatch

```
https://github.com/NixOS/nixpkgs/blob/fc861d3e06657c4a43c958333f8aaa812b359649/doc/languages-frameworks/go.section.md?plain=1#L22

---

Thank you! :)